scripts/split-e2e-tests.ts

```diff
@@ -471,17 +478,19 @@ function main(): void {
   );
   const splitV4MigrationTests = splitTests(
```

Comment:
nit: rename function

```suggestion
  const splitV5MigrationTests = splitTests(
```